setup.cfg

```diff
@@ -70,7 +71,7 @@ install_requires =
 
 [options.extras_require]
 pyside2 =
-    PySide2>=5.12.3,!=5.15.0
+    PySide2>=5.13.2,!=5.15.0
```

Comment:
Is this necessary? e.g. conda-forge pyqt is still 5.12.3.

Reply:
Ah, just saw the discussion above. Hmm. Well, I guess it's fine since on conda-forge we depend on pyqt... 🤷